service/src/main/java/com/github/szsalyi/flashcard/service/user/UserVOtoUserEntityConverter.java

```diff
@@ -0,0 +1,24 @@
+package com.github.szsalyi.flashcard.service.user;
+
+import com.github.szsalyi.flashcard.entity.UserEntity;
+import com.github.szsalyi.flashcard.service.mapper.common.Mapper;
+import org.dozer.DozerBeanMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserVOtoUserEntityConverter implements Converter<UserVO, UserEntity> {
+
+    @Autowired
+    private static DozerBeanMapper mapper = Mapper.getDozerBeanMapper();
```

Comment:
The point of dependency injection is that it's not you who is creating/getting an instance of an object, but some kind of other tool (like a framework) will give it to you.

In this situation there is no point of using the `Autowired` annotation since you will be requesting a `DozerBeanMapper` object from the `Mapper` class.

Create a Spring bean instance from the `DozerBeanMapper` class in a Spring configuration class and autowire that instance.

You don't need the `Mapper` class.